integrationTests/cypress/integration/redmine_administration_rest.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Administrative Procedures via API
+
+  @requires_testuser
+  Scenario: cas user (admin) => access users.json with api key => success
+    Given the user is member of the admin user group
+    And the user has an internal default redmine account
+    When the user request the user.json from Redmine via API key
+    Then the user receives the user.json as response
+
+  @requires_testuser
+  Scenario: cas user => access users.json with api key => unauthorized
+    Given the user is not member of the admin user group
+    And the user has an internal default redmine account
+    When the user request the user.json from Redmine via API key
+    Then the user receives an unauthorized access response
+
+  @requires_testuser
+  Scenario: cas user + internal special redmine admin account => access users.json with api key => success
```

Comment:
Suggestion:
"cas user + internal special redmine admin account => access users.json with api key => success" -->
"a regular CAS user with an exisiting redmine admin account access the users api endpoint successfully"